FlaxEditor/Windows/GameWindow.cs

```diff
@@ -287,6 +289,16 @@ public override void Draw()
                     Render2D.DrawRectangle(new Rectangle(new Vector2(4), Size - 8), Color.Orange * alpha);
                 }
             }
+            
+            Editor.Instance.SceneEditing.Selection.ForEach((node) =>
+            {
+                if (node.EditableObject is UIControl controlActor)
+                {
+                    if (controlActor.Control != null)
+                        Render2D.DrawRectangle(controlActor.Control.Bounds, Editor.Instance.Options.Options.Visual.SelectionOutlineColor0, 2f);
```

Comment:
Hold up, is it just me or are you not using the `UISelectionOutlineSize` here?

Reply:
Omg, fixed.